examples_tests/15.LayeredCubeMapShadows/deriv_map_gen.comp

```diff
@@ -1,97 +1,46 @@
 #version 450 core
 
-//place for #defines to put with sprintf
-%s
+layout (local_size_x = 16, local_size_y = 16) in;
 
-#if 0
-#extension GL_NV_compute_shader_derivatives : enable
-#if defined(GL_NV_compute_shader_derivatives)
-layout (derivative_group_quadsNV) in;
+layout (binding = 7) uniform sampler2D bumpMapSampler;
+layout (binding = 0, rg16_snorm) uniform image2D derivativeMapImage;
 
-#define IRR_DDX_IS_DEFINED 1
-#define IRR_DDY_IS_DEFINED 1
+layout (location = 0) uniform float uHeightScaleFactor = 0.001;
 
-float DDX(in float x)
-{
-	return dFdx(x);
-}
+shared float smem[324];//18*18
```

Comment:
use a macro to make it configurable

Reply:
smem size? But why?

Reply:
one day when all devices support 1024-sized work groups....